docs/source/getting-started/examples/histograms.ipynb

```diff
@@ -269,7 +370,7 @@
    "name": "python",
    "nbconvert_exporter": "python",
    "pygments_lexer": "ipython3",
-   "version": "3.11.7"
+   "version": "3.13.2"
   }
  },
  "nbformat": 4,
```

Comment:
Would kind of prefer not to touch this notebook in this PR, if it's not essential, but that may not be possible.